CONTRIBUTING.md

```diff
@@ -351,36 +351,59 @@ Read more about [Stencil Unit testing](https://stenciljs.com/docs/unit-testing).
 
 ### End-to-end / integration tests
 
-Stencil provides utility functions from [Jest](https://jestjs.io/) and [Puppeteer](https://pptr.dev/) to create end-to-end / integration tests. End-to-end / integraton testing makes it easier to test multiple components within a real browser. Each component includes a `<component>.e2e.ts` file.
+- e2e tests and how to use selectors etc
+- where to put the tests
+- a11y tests
+- visual tests
+
+
+[Cypress](https://www.cypress.io/) is the testing framework used to run end-to-end / intergration tests. Within the `react/src/component-tests` directory is a folder for each component that includes a `<component>.cy.tsx` file along with any helper data for the test.

```

Comment:
I would like to see a bit more depth on the file structure - we've got a pattern with tests in component.cy.tsx, componentConstants.tsx to store values for testing against, and componentTestData.tsx for storing code snippets to improve test readability.